algorithms/CPlusPlus/Trees/print-all-nodes-at-a-level.cpp

```diff
@@ -0,0 +1,80 @@
+/* Description - We have to print all nodes at a level 'k' of the tree.
+
+For example- If we are given the following tree, the nodes at level 1 are 1, nodes at level 2 are 2,3, nodes at level 3 are 7, 9 and nodes at level 4 are 21 and 11.
+(Note- The level starts from 1, i.e root is at level 1)
+
+            1          level 1
+          /   \
+         2     3       level 2
+      	      /  \    
+             7    9    level 3
+	    /      \
+	   21      11  level 4
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+typedef struct Node
+{
+	int data;
+	struct Node* left;
+	struct Node* right;
+	
+	Node(int val)
+	{
+		data= val;
+		left=right=NULL;
+	}
+}node;
```

Comment:
In c++ there's no difference between `struct` and `typedef struct` , since unlike C in C++ the `struct` keyword is optional which makes it effectively same as `typedef struct` in C.
```suggestion
struct Node
{
	int data;
	struct Node* left;
	struct Node* right;
	
	Node(int val)
	{
		data= val;
		left=right=NULL;
};
```

Reply:
Won't I have to write struct Node everywhere instead of just node then?